recipes/simpleitk/bld.bat

```diff
@@ -0,0 +1,49 @@
+mkdir C:\b
+cd C:\b
+
+set "CMAKE_GENERATOR=Visual Studio 9 2008"
+if "%PY_VER%" == "3.3" set "CMAKE_GENERATOR=Visual Studio 10"
+if "%PY_VER%" == "3.4" set "CMAKE_GENERATOR=Visual Studio 10"
+if "%PY_VER%" == "3.5" set "CMAKE_GENERATOR=Visual Studio 14"
+
+if "%ARCH%" == "64" set "CMAKE_GENERATOR=%CMAKE_GENERATOR% Win64"
```

Comment:
So, I think `conda-build` sets this for you. Is that right, @patricksnape? I ask you as I thought you added it. 😄